### PR TITLE
Setup Sendgrid

### PR DESCRIPTION
**Pull request by cantino**

### CODE REVIEW

File: app/mailers/provider_mailer.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class ProviderMailer < ApplicationMailer
   def request_created_email
     @provider = params[:linked_token].entity.provider
-    @request_url = 'localhost:3000/' + 'request/' + params[:linked_token].entity.id.to_s
-    @edit_url  = 'localhost:3000/' + 'provider/' + params[:linked_token].token
+    @request_url = HOST_AND_SCHEME + '/request/' + params[:linked_token].entity.id.to_s
```
Comment (tuntisz) [inline app/mailers/provider_mailer.rb:4]: Thanks for this. Is HOST_AND_SCHEME a rails thing?

File: config/initializers/1env.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Starts with 1 so that it loads first.
+
+HOST_AND_SCHEME = ENV['HOST_AND_SCHEME'].presence || 'http://localhost:3000'
```
Comment (tuntisz) [inline config/initializers/1env.rb:3]: Ignore the other comment, just saw this.

**Review by tuntisz — APPROVED** [review]
Nice! Looks good.

File: app/graphql/mutations/create_provider_and_request.rb [hunk 3]
```diff
@@ -41,8 +41,7 @@ def resolve(first_name:, last_name: "", neighborhood: "", city:, state:, email:,
       provider.save!
       request.save!
       linked_token = LinkCreator.create_token(request)
-      # TODO: Do we plan on using Action Job? If so, make async
-      ProviderMailer.with(linked_token: linked_token).request_created_email.deliver_now
+      ProviderMailer.with(linked_token: linked_token).request_created_email.deliver_later
```
Comment (cantino) [inline app/graphql/mutations/create_provider_and_request.rb:44]: @tuntisz Added an in-process background worker for now. It's not a long term solution— we should move to sidekiq eventually.

**Pull request merged**